pfcpiface/conn.go

```diff
@@ -63,6 +65,80 @@ type PFCPConn struct {
 	shutdown chan struct{}
 
 	metrics.InstrumentPFCP
+	hbMonitor   *hbMonitor
+	pendingReqs sync.Map
+}
+
+type HbTimerStatus uint8
+
+const (
+	CancelHbTimer HbTimerStatus = iota
+	StartHbTimer
+	StopHbTimer
+)
+
+type hbMonitor struct {
```

Comment:
can this be replaced with a derived context? 

pConn.hbctx, pConn.hbCancel = context.WithCancel(pConn.ctx)

So that in startHeartBeatMonitor we can select on pConn.hbCtx.Done() which will notify us if either the Conn itself got shutdown or just HB monitor got cancelled

Reply:
we can get rid of CancelHeartBeatTimer